modules/monitoring-enabling-the-platform-alertmanager-instance-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-alert-routing-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-enabling-the-platform-alertmanager-instance-for-user-defined-projects_{context}"]
+= Enabling the platform Alertmanager instance for user-defined projects
```

Comment:
should it be "Enabling the platform Alertmanager instance for user-defined alert routing"?

Reply:
Revised per your suggestion